scripts/barclamp_mgmt_lib.rb

```diff
@@ -179,7 +179,7 @@ def prepare_navigation(hash, breadcrumb, indent, level)
       options_string.prepend(", ") unless options_string.empty?
 
       if values.keys.empty?
-        result.push "level#{level}.item :#{key}, t(\"nav.#{current_path.join(".")}\"), #{link}#{options_string}".indent(indent)
+        result.push "level#{level}.item :#{key}, t(\"nav.#{current_path.join(".")}.title\"), #{link}#{options_string}".indent(indent)
```

Comment:
Metrics/LineLength: Line is too long. [133/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)